### PR TITLE
Add KiRoWi2021 entry

### PR DESCRIPTION
**Pull request by mathmerizing**

### CODE REVIEW

**Review by bangerth — APPROVED** [review]
Nice!

File: publications-2021.bib [hunk 1]
```diff
@@ -1,5 +1,19 @@
 % Encoding: US-ASCII
 
+@article{KiRoWi2021,
+	title = {Geometric multigrid with multiplicative Schwarz smoothers for eddy-current and Maxwell’s equations in deal.II},
```
Comment (bangerth) [inline publications-2021.bib:4]: You'll need an ASCII apostrophe:
```suggestion
	title = {Geometric multigrid with multiplicative Schwarz smoothers for eddy-current and Maxwell's equations in deal.II},
```

**Review by bangerth — CHANGES_REQUESTED** [review]

**Review by bangerth — APPROVED** [review]

File: publications-2021.bib [hunk 2]
```diff
@@ -1,5 +1,19 @@
 % Encoding: US-ASCII
 
+@article{KiRoWi2021,
+	title = {Geometric multigrid with multiplicative Schwarz smoothers for eddy-current and Maxwell's equations in deal.II},
+	journal = {Examples and Counterexamples},
+	volume = {1},
+	pages = {100027},
+	year = {2021},
+	issn = {2666-657X},
+	doi = {https://doi.org/10.1016/j.exco.2021.100027},
```
Comment (bangerth) [inline publications-2021.bib:10]: Looking through this again, this isn't actually the DOI. The DOI should look like this:
```suggestion
	doi = {10.1016/j.exco.2021.100027},
```

**Review by bangerth — COMMENTED** [review]
Thanks. Do you know how to squash commits into one?

File: publications-2021.bib (hunk 2 above)
Comment (mathmerizing) [reply publications-2021.bib:10]: I exported the Bibtex citation from ScienceDirect. That's why I am surprised that the journal returns a wrong Bibtex entry. But I can commit your change to the DOI and try to squash all the commits.

**Comment by mathmerizing** [comment]
> Thanks. Do you know how to squash commits into one?

Thank you very much Prof. Bangerth for your help! I just squashed the commits and I think that everything should be fine now.

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
Thanks a lot!

**Pull request merged**